docs/README.md

```diff
@@ -47,6 +47,15 @@ The following CSI interfaces are implemented:
  * The uid/gid configured on the access point is either the uid/gid specified in the storage class, a value in the gidRangeStart-gidRangeEnd (used as both uid/gid) specified in the storage class, or is a value selected by the driver is no uid/gid or gidRange is specified.
  * We suggest using [static provisioning](https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/master/examples/kubernetes/static_provisioning/README.md) if you do not wish to use user identity enforcement.
 
+**Note**
+
+If you want to pass any other mountOptions to EFS CSI driver while mounting, they can be passed in pv and storage class for static and dynamic provisioning respectively.
+Examples of some mountOptions that can be passed:
+
+**lookupcache**: Specifies how the kernel manages its cache of directory entries for a given mount point. Mode can be one of all, none, pos, or positive. Each mode has different functions and for more information you can refer to this [link](https://linux.die.net/man/5/nfs).
+
+**iam**: The IAM option allows you to control the AWS Identity and Access Management Permissions for the EFS CSI driver and the associated EFS file system.
```

Comment:
I would reword the `iam` option. Maybe something like

"Use the CSI Node Pod's IAM identity to authenticate with EFS".